src/Components/UI/input/MyInput.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const MyInput = (props) => {
+  return (
+    <input {...props} />
+  )
+}
```

Comment:
This component does nothing but returns other component with the same set of properties... it serves as just a wrapper without any benefits, thus we don't need it